From: Jan Engelhardt <jengelh@inai.de>
Date: 2013-01-04 19:25:20.767135850 +0100

xm: avoid pointless double assignment

gcc:
 Color.c: In function 'GetColors':
 Color.c:1004:60: warning: operation on 'new_colors.top_shadow.pixel' may be undefined [-Wsequence-point]
 Obso2_0.c:963:60: warning: operation on 'new_colors.top_shadow.pixel' may be undefined [-Wsequence-point]
rpmlint:
 W: motif sequence-point Color.c:1004
 W: motif sequence-point Obso2_0.c:963

---
 lib/Xm/Color.c   |    2 +-
 lib/Xm/Obso2_0.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: motif-2.3.4/lib/Xm/Color.c
===================================================================
--- motif-2.3.4.orig/lib/Xm/Color.c
+++ motif-2.3.4/lib/Xm/Color.c
@@ -1002,7 +1002,7 @@ GetColors(
 	 * implementations of X).
 	 */
 	new_colors.foreground.pixel = new_colors.top_shadow.pixel = 
-		new_colors.top_shadow.pixel = new_colors.select.pixel = 0;
+		new_colors.select.pixel = 0;
 
 	/*  Generate the foreground, top_shadow, and bottom_shadow based  */
 	/*  on the background                                             */
Index: motif-2.3.4/lib/Xm/Obso2_0.c
===================================================================
--- motif-2.3.4.orig/lib/Xm/Obso2_0.c
+++ motif-2.3.4/lib/Xm/Obso2_0.c
@@ -961,7 +961,7 @@ _XmGetColors(
 	 * implementations of X).
 	 */
 	new_colors.foreground.pixel = new_colors.top_shadow.pixel = 
-		new_colors.top_shadow.pixel = new_colors.select.pixel = 0;
+		new_colors.select.pixel = 0;
 
 	/*  Generate the foreground, top_shadow, and bottom_shadow based  */
 	/*  on the background                                             */
